New landing page (#370)
rescript-association/rescript-lang.org: _blogposts/2021-06-25-roadmap-2021-and-new-landing-page.mdx

---
author: rescript-team
date: "2021-06-25"

> title: Roadmap & New Landing Page

Reviewer: it reads like
```
Roamdap & New
Landing Page
```
asymetric?

Author: should we go with `Roadmap 2021 & New Landing Page`?

Author: should be more symmetric now

description: |
  Announcing our new roadmap for 2021 / 2022, release cycle plans and landing page.
---

import Image from "src/components/Image.mjs"

## Team Update

It has almost been a year since we originally [launched our new ReScript brand](/blog/bucklescript-is-rebranding), and we think it has been a pretty successful journey so far:

> - An all new documentation experience

Reviewer: ```suggestion
- Brand new documentation experience
```

Let's get rid of "An"

- Relaunch of `rescript-react` and its new docs
- Release of our re-imagined VSCode & Vim plugins
- New syntax, compiler and build system improvements, including incremental build performance at scale

> - Improved polymorphic variant interop for numbers and strings

Reviewer: ```suggestion
- Improved interop with polymorphic variants for numbers and strings
```
the word "improved" was used twice, also could be a bit more lightweight.

> - A greatly simplified object system (no need for `Js.t`)

Reviewer: ```suggestion
- Accessible object system (no need for `Js.t`)
```

I don't like these sentences starting with "A". (Very subjective feedback haha)

- Making every part of ReScript fully Open Source and therefore independent from any company politics
- The release of our new `rescript` npm package and cli to replace `bs-platform`
- etc. 

> Of course there are many more things to come in the near future, and we wanted to make sure to keep track of our main goals in a more reliable and transparent way.

Reviewer: ```suggestion
Stay tuned, this is just the beginning!
```

The second part of the sentence is a bit confusing, so I left it out.

### New Roadmap and Release Cycle Plans

We had some thorough discussions about the future of the project and outlined the most important milestones for the next upcoming releases.

> **We also settled on a release cycle plan to give a better overview on estimated releases:**

Reviewer: This sentence also starts with "We", just as the previous one.

```suggestion
**Here's the gist:**
```

- Two release channels: `stable` and `experimental`
- More predictable release dates and better migration steps
- Better communication and discussion for breaking changes

> - Helps the community / companies to align on major stable versions

Reviewer: ```suggestion
- Help the community / companies align on major stable versions
```

The detailed roadmap with all our planned changes (and definition of our release cycle plan) can be found in our [community / roadmap](/community/roadmap) section.

### New Landing Page

> After several iterations, and lots of desperate days of designing and copywriting, we are happy to announce that we are finally releasing our new landing page.

Reviewer: ```suggestion
After several iterations, we are happy to announce our new [landing page](https://www.rescript-lang.org).
```
